spyderplugins/p_breakpoints/breakpoints.py

```diff
@@ -14,10 +14,10 @@
 
 # Local imports
 from spyderlib.baseconfig import get_translation
-_ = get_translation("p_breakpoints", dirname="spyderplugins")
+_ = get_translation("p_breakpoints", dirname="spyderplugins.p_breakpoints")

```

Comment:
There is something strange here, dirname should be set to the module name, not to the directory name. (Same problem for all 3 plugins.) Should I change get_translation ? This will be quite confusing for future plugin devs.